const createError = require("../utils/createError");
exports.createEducation = async () => {
  try {
  } catch (error) {
    next(error);
  }
};

exports.updateEducation = async () => {
  try {
  } catch (error) {
    next(error);
  }
};
exports.deleleEducation = async () => {
  try {
  } catch (error) {
    next(error);
  }
};
